.github/workflows/acceptance.yml

```diff
@@ -6,6 +6,32 @@ on:
   pull_request:
 
 jobs:
+    getsentry:
+      runs-on: ubuntu-16.04
+      steps:
+        - name: getsentry token
+          id: getsentry
+          uses: getsentry/action-github-app-token@v1
+          with:
+            private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
+
+        # Notify getsentry
+        - name: Dispatch getsentry tests
+          uses: actions/github-script@v3
+          with:
+            github-token: ${{ steps.getsentry.outputs.token }}
```

Comment:
The only other way afaik, is to use a personal access token that is bound to my account (or I suppose a bot account). Instead we use GitHub apps for auth as it is easier to manage since the `getsentry` org can control access.

Reply:
We've been using a PAT created from the getsentry-bot account for OSS releases and it is working well. If GitHub apps are working out well for you, I think this is fine tho.